examples/dag/simple_dag.py

```diff
@@ -154,6 +154,42 @@
     ).depends_on(openai_3_node)
     wf.flow.add_nodes(openai_4_node)
 
+    # OpenAI node 5 depending on OpenAI node 3 and OpenAI node 4 and used custom inputs and transformations
+    def merge_and_short_content(inputs: dict, outputs: dict[str, dict]):
```

Comment:
I can test such behaviour with more complex example in your branch?

Reply:
yes